### ISSUE TITLE
New integration tests: checks for component and analysis results stored in the S3

### ISSUE THREAD

**Issue by tisnik** [openshiftio/openshift.io#1077]
Having the new functionality implemented via #954 it would be nice to develop following new tests:

### Pypi ecosystem
- [ ] analysis for 'simple' package (no dependents)
- [ ] analysis for more complicated package (complicated == library with many dependents etc.)
- [ ] analysis for package with more licenses

### NPM ecosystem
- [ ] analysis for 'simple' package (no dependents)
- [ ] analysis for more complicated package (complicated == library with many dependents etc.)
- [ ] analysis for package with more licenses

### Maven ecosystem
- [ ] analysis for 'simple' package (no dependents)
- [ ] analysis for more complicated package (complicated == library with many dependents etc.)
- [ ] analysis for package with more licenses

**Comment by msrb** [comment]
@tisnik moving to the current sprint. Feel free to move it back log if we don't want to do this now.

**Comment by sivaavkd** [comment]
@tisnik  Where are we with this ?  Is this spilling over to 148 ?  Please update accordingly. cc @msrb

**Comment by tisnik** [comment]
depends on #3605

**Comment by msrb** [comment]
@tisnik please close this issue, it it is finished, or move it to the current sprint/to the backlog. Thanks :wink:

**Comment by GeorgeActon** [comment]
@tisnik @msrb  Is this still relevant? If so it should be moved to planner back log.
if it is already done labels should be updated andit should be closed.

**Comment by GeorgeActon** [comment]
@tisnik What is the actual status of this?
